takwimu/profiles/census.py

```diff
@@ -139,8 +142,8 @@ def get_profile(geo, profile_name, request):
         data['demographics'] = get_demographics(geo, session, country, level)
         data['elections'] = get_elections(geo, session)
         data['crops'] = get_crop_production(geo, session, country, level)
-        data['health_centers'] = get_health_centers(
-            geo, session, country, level)
+        # data['health_centers'] = get_health_centers(
+        #     geo, session, country, level)
```

Comment:
Why are we removing these?

Reply:
@kilemensi working on fixing the health centers, they had an issue that was causing everything to fail